lua/mini/ai.lua

```diff
@@ -1562,24 +1562,31 @@ H.get_matched_ranges_builtin = function(captures)
   -- Get parser (LanguageTree) at cursor (important for injected languages)
   local pos = vim.api.nvim_win_get_cursor(0)
   local lang_tree = parser:language_for_range({ pos[1] - 1, pos[2], pos[1] - 1, pos[2] })
-  local lang = lang_tree:lang()
-
-  -- Get query file depending on the local language
-  local query = vim.treesitter.query.get(lang, 'textobjects')
-  if query == nil then H.error_treesitter('query', lang) end
```

Comment:
There still needs to be some kind of error indicating that there is no query available for language(s). Probably needs to keep track of traversed languages and show error if all of them don't have dedicated queries.